src/renderer/themes.css

```diff
@@ -1041,7 +1042,7 @@ body[dir='rtl'] {
   --horizontal-directionality-coefficient: -1;
 }
 
-body[dir='rtl'] [data-prefix="fas"]:not([data-icon="magnifying-glass"]):not([data-icon="circle-question"]) {
+body[dir='rtl'] [data-prefix="fas"]:not([data-icon="magnifying-glass"], [data-icon="circle-question"]) {
```

Comment:
Is this correct?

Reply:
Yes, just re-validated with Arabic and Hebrew. Peep the closing parenthesis location

Reply:
Ah ok it's quite difficult to read with a long line & parenthesis...